docker-compose-dev.yaml

```diff
@@ -0,0 +1,32 @@
+### INSTRUCTIONS ###
+## The purpose of this docker container is to simplify bringing up Gaps while developing.
+# 1. Build Gaps Jar
+#     - mvn clean install
+# 2. Bring up development container in background. This will build a docker image if it does not already exist.
+#     - docker-compose -f docker-compose-dev.yaml up -d
+#
+# Optional: Uncomment & override env variables if needed
+#
+## To test any new changes do the following
+#     - build new jars
+#     - restart docker container; a rebuild is no longer necessary
+
+version: "3"
+services:
+  gaps:
+    container_name: gaps_dev
+    build:
+      context: .
+      dockerfile: Dockerfile.dev
+    environment:
+      #gapsVersion: *
```

Comment:
With these three values commented, docker-compose says the file is invalid and won't run. I had to uncomment to run so we should probably just leave them uncommented. I do a find/replace on version updates so I will be able to catch and update this as gaps progresses.

Reply:
This should be resolved with latest commit. Bad clean-up job on my part.

Reply:
Good stuff